PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutVaultRequest.java

```diff
@@ -14,7 +14,25 @@ public class PayPalNativeCheckoutVaultRequest extends PayPalNativeRequest implem
 
     private boolean shouldOfferCredit;
 
+    /**
+     * Deprecated. Use {@link PayPalNativeCheckoutVaultRequest#PayPalNativeCheckoutVaultRequest(boolean)} instead.
+     */
+    @Deprecated
     public PayPalNativeCheckoutVaultRequest() {
+        this(false);
+    }
+
+    /**
+     * @param hasUserLocationConsent is an optional parameter that informs the SDK
```

Comment:
```suggestion
     * @param hasUserLocationConsent informs the SDK
```

Removing optional since in this constructor it's technically not optional

Reply:
Good catch!